apps/teams/tests/test_permissions.py

```diff
@@ -65,6 +65,7 @@ def test_content_types():
     "template_partials",
     "tz_detect",
     "users",
+    "utils_tests",
```

Comment:
I'm not 100% certain why this only started failing here, or if this is valid...

There is indeed an app with this label: 

https://github.com/dimagi/open-chat-studio/blob/45c09ef80c7dae4f3b5da7cdc567ddaa4552d6b5/apps/utils/tests/conftest.py#L5